index.html

```diff
@@ -5,8 +5,165 @@
     <title>HTML Academy: Седона</title>
   </head>
   <body>
+    <!--Header-->
+    <header class="page-header">
+      <!--Навигация-->
+      <nav class="navigation">
+        <!--Логотип-->
+        <a href="index.html">
+          <img src="" alt="Логотип Седоны">
+        </a>
+        <!--Основная навигация по сайту-->
+        <ul class="navigation-list">
```

Comment:
Необязательно, но так можно подписать прям списки

было
![image](https://user-images.githubusercontent.com/47776594/150673862-4995c3d9-75fc-4340-b26f-e09dbc3dbe58.png)
Станет
```suggestion
        <ul class="navigation-list" aria-label="Основная навигация">
```
![image](https://user-images.githubusercontent.com/47776594/150673904-0c7dcd10-818e-4b9c-8cae-1ac32ad1383e.png)